tools/Google.Cloud.Tools.VersionCompat/Program.cs

```diff
@@ -66,24 +70,78 @@ public static DiffResult Check(AssemblyDefinition older, AssemblyDefinition newe
             }
         }
 
-        static AssemblyDefinition Load(string name)
+        /// <summary>
+        /// Loads a package from NuGet and extracts an assembly definition.
+        /// </summary>
+        /// <param name="package">The name of the package to load. Must not be null.</param>
+        /// <param name="version">The version of the package to load. May be null, in which case the latest stable version is downloaded.</param>
+        /// <param name="tfm">The target framework to find within the package. May be null, in which case "netstandard2.0" is assumed.</param>
+        /// <param name="assemblyName">The name of the assembly to find within the package. May be null, in which case it's assumed to be the same as the package.</param>
+        /// <returns></returns>
+        public async static Task<AssemblyDefinition> LoadPackageAsync(string package, string version, string tfm, string assemblyName)
+        {
+            var client = new HttpClient();
+            // Handily, this automatically loads the latest stable release if version is null.
+            var url = $"https://www.nuget.org/api/v2/package/{package}/{version}";
+            var response = await client.GetAsync(url);
+            response.EnsureSuccessStatusCode();
+            var bytes = await response.Content.ReadAsByteArrayAsync();
+
+            string expectedPath = $"lib/{tfm}/{assemblyName}.dll";
```

Comment:
I expect this will fail if a tfm isn't given.
Which doesn't matter for now, but should be fixed if specifying a tfm really is optional.

Reply:
Fixed - both `tfm` and `assemblyName` weren't properly optional.